### ISSUE TITLE
Orbits around bodies without predefined reference frame do not have str representation

### ISSUE THREAD

**Issue by Juanlu001** [poliastro/poliastro#476]
_Comes from #474, by @TimothySHamilton_

For example, for any orbit around `Moon`:

```pytb
In [32]: orbit_moon_enter = Orbit.from_vectors(Moon, r, v)

In [33]: orbit_moon_enter
Out[33]: 
In [34]: str(orbit_moon_enter)
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
~/.miniconda36/envs/poliastro37/lib/python3.7/site-packages/poliastro/frames.py in get_frame(attractor, plane, obstime)
    242     try:
--> 243         frames = _FRAME_MAPPING[attractor]
    244     except KeyError:

KeyError: Moon (☾)

During handling of the above exception, another exception occurred:

NotImplementedError                       Traceback (most recent call last)
<ipython-input-34-12bb16b34e33> in <module>()
----> 1 str(orbit_moon_enter)

~/.miniconda36/envs/poliastro37/lib/python3.7/site-packages/poliastro/twobody/orbit.py in __str__(self)
    357         return ORBIT_FORMAT.format(
    358             r_p=self.r_p.to(unit).value, r_a=self.r_a.to(unit), inc=self.inc.to(u.deg),
--> 359             frame=self.frame.__class__.__name__,
    360             body=self.attractor,
    361             epoch=self.epoch, scale=self.epoch.scale.upper(),

~/.miniconda36/envs/poliastro37/lib/python3.7/site-packages/poliastro/twobody/orbit.py in frame(self)
     86         """
     87         if self._frame is None:
---> 88             self._frame = get_frame(self.attractor, self._plane, self.epoch)
     89 
     90         return self._frame

~/.miniconda36/envs/poliastro37/lib/python3.7/site-packages/poliastro/frames.py in get_frame(attractor, plane, obstime)
    243         frames = _FRAME_MAPPING[attractor]
    244     except KeyError:
--> 245         raise NotImplementedError("Frames for orbits around custom bodies are not yet supported")
    246 
    247     try:

NotImplementedError: Frames for orbits around custom bodies are not yet supported
```

This should be easy to fix, having two templates for the string representation and falling back to a simpler one when there is no frame. And then, actually implementing frames for all sorts of bodies: https://github.com/poliastro/poliastro/issues/109.

**Comment by AntoniiaK** [comment]
Hello! 
Do you want remove these exceptions
   ```
 try:
        frames = _FRAME_MAPPING[attractor]
    except KeyError:
        raise NotImplementedError("Frames for orbits around custom bodies are not yet supported")

    try:
        frame_class = frames[plane]
    except KeyError:
        raise NotImplementedError(
            "A frame with plane {} around body {} is not yet implemented".format(plane, attractor))

```

And get:
![plar](https://user-images.githubusercontent.com/45198538/50015027-956f8c80-ffd6-11e8-891c-554222790c9a.jpg)

**Comment by Juanlu001** [comment]
@AntoniiaK rather than removing them, I was thinking of chaning this code:

https://github.com/poliastro/poliastro/blob/d5d0ad1218c075b0d007e9d386f52dc628419f51/src/poliastro/twobody/orbit.py#L351-L362

Into something like:

```
try:
    return ORBIT_FORMAT.format( 
         r_p=self.r_p.to(unit).value, r_a=self.r_a.to(unit), inc=self.inc.to(u.deg), 
         frame=self.frame.__class__.__name__, 
         body=self.attractor, 
         epoch=self.epoch, scale=self.epoch.scale.upper(), 
    )
except NotImplementedError:
    return ORBIT_NOFRAME_FORMAT.format( 
         r_p=self.r_p.to(unit).value, r_a=self.r_a.to(unit), inc=self.inc.to(u.deg), 
         body=self.attractor, 
         epoch=self.epoch, scale=self.epoch.scale.upper(), 
    )
```

so we can still raise an appropriate error when the user tries to get a frame that is not implemented yet.

**Comment by Juanlu001** [comment]
Closed in #506.
